### PR TITLE
Merge back release 3 6 develop

### PR DESCRIPTION
**Pull request by phpgeekfr**
Merge back release 3 6 develop

### CODE REVIEW

File: apps/catalog/catalog/src/app/landing/landing.component.html [hunk 1]
```diff
@@ -11,56 +11,78 @@ <h2>for TV, VOD and ancillary rights</h2>
...
+  <landing-content>
+    <section class="landing-presentation max-width">
+      <h3 class="mat-display-1">Deals made easy</h3>
+      <p class="mat-body-1">Archipel Market is an ongoing film market platform. <br>
+        It facilitates interactions between sales agents and buyers all year round. </p>
+      <mat-tab-group animationDuration="1000ms" selectedIndex="0" (selectedTabChange)="onTabChanged($event)">
+        <mat-tab [label]="tabNames[0]">
+          <header fxLayout fxLayout.lt-lg="column" fxLayoutGap="32px">
+            <video #sellerVideo [muted]="true" autoplay loop src="assets/videos/sellerLP.mp4" type="video/mp4"></video>
+            <div>
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/landing/landing.component.html:23]: You don't need this div

File: apps/catalog/catalog/src/app/landing/landing.component.html [hunk 2]
```diff
@@ -11,56 +11,78 @@ <h2>for TV, VOD and ancillary rights</h2>
...
-            <h3>{{ tabTitle[1].title }}</h3>
-            <ng-container *ngIf="tabTitle[1].description as description">
-              <p class="mat-body-1">{{ description }}</p>
-            </ng-container>
+        </mat-tab>
+
+        <mat-tab [label]="tabNames[1]">
+           <header fxLayout fxLayoutGap="48px" fxLayout.lt-lg="column" fxLayoutGap="32px">
+            <video #buyerVideo [muted]="true" autoplay loop src="assets/videos/buyerLP.mp4" type="video/mp4"></video>
+            <div>
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/landing/landing.component.html:43]: same

File: apps/catalog/catalog/src/app/landing/landing.component.html [hunk 3]
```diff
@@ -11,56 +11,78 @@ <h2>for TV, VOD and ancillary rights</h2>
...
+      <p>We take 10% commission when there is a deal</p>
+      <a mat-flat-button color="primary" routerLink="/auth/identity">
+        <span>Join now</span>
         <mat-icon svgIcon="arrow_forward"></mat-icon>
       </a>
     </section>
+
+    <section class="expertise max-width">
+      <h3 class="mat-display-1">Our expertise</h3>
+      <div>
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/landing/landing.component.html:73]: same

File: apps/catalog/catalog/src/app/landing/landing.component.html [hunk 4]
```diff
@@ -11,56 +11,78 @@ <h2>for TV, VOD and ancillary rights</h2>
...
-    <section fxLayout="column" fxLayoutAlign="start center" class="join" fxLayoutGap="40px">
-      <p class="mat-display-2">Interested? Join now!</p>
-      <a mat-flat-button color="accent" routerLink="/auth/identity">
-        <span>Sign up</span>
+        </mat-tab>
+      </mat-tab-group>
+    </section>
+    
+    <section fxLayout="column" fxLayoutAlign="start center" class="dark-contrast-theme join" fxLayoutGap="40px">
+      <p class="mat-display-2">Subscribe for free!</p>
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/landing/landing.component.html:63]: maybe use a heading if it's mat-display-2

File: apps/festival/festival/src/app/event/login/login.component.html [hunk 5]
```diff
@@ -36,7 +36,9 @@ <h1>Welcome to Archipel Market!</h1>
 
     <footer fxLayout="column" fxLayoutGap="16px">
       <a routerLink="/auth/reset-password" color="primary">Forgot your password ?</a>
-      <button mat-flat-button color="primary" type="submit">{{ buttonText }}</button>
+      <div>
```
Comment (GrandSchtroumpf) [inline apps/festival/festival/src/app/event/login/login.component.html:39]: same, use align-self or justify-self to have specific behavior on the button

File: apps/festival/festival/src/app/event/view/view.component.ts [hunk 6]
```diff
@@ -33,6 +35,13 @@ export class EventViewComponent implements OnInit {
     private authService: AuthService
   ) { }
 
+  @HostListener('window:popstate', ['$event'])
+  onPopState() {
+    if (!this.preventBrowserEvent) {
+      this.goBack();
```
Comment (GrandSchtroumpf) [inline apps/festival/festival/src/app/event/view/view.component.ts:41]: stay consistant. Either it's `clickBack` or `goBack`

File: apps/festival/festival/src/app/landing/landing.component.scss [hunk 7]
```diff
@@ -39,7 +36,26 @@ landing-content {
...
   .landing-presentation {
-    padding: 80px
+    padding: 80px;
+    mat-tab-group {
+      padding-inline: 80px;
+      mat-tab-header {
+        position: sticky;
+        top: 0;
+        background-color: var(--background);
+        z-index: 1000;
```
Comment (GrandSchtroumpf) [inline apps/festival/festival/src/app/landing/landing.component.scss:46]: Use a normal index number, like 1 or 2

File: apps/festival/festival/src/app/marketplace/title/list/list.component.html [hunk 8]
```diff
@@ -1,4 +1,4 @@
-<list-page [items]="movies$ | async">
+<list-page [items]="movies$ | async" [pdfExport]="true">
```
Comment (GrandSchtroumpf) [inline apps/festival/festival/src/app/marketplace/title/list/list.component.html:1]: Would be better to use the `@boolean` decorator of the list-page to coerc the bool so you can do
```html
<list-page [items]="movies$ | async" pdfExport>
```

File: libs/landing/src/lib/shell/shell.component.html [hunk 9]
```diff
@@ -105,8 +105,10 @@ <h3>Schedule a demo with our team!</h3>
       <section class="max-width" fxLayout="column" fxLayoutAlign="start center" fxLayoutAlign.lt-md="center center"
         fxLayoutGap="32px">
         <div fxLayout="row wrap">
-          <app-logo fxLayout fxFlex></app-logo>
-          <form fxFlex [formGroup]="newslettersForm" (ngSubmit)="subscribeToNewsletters(newslettersForm)" >
+          <div fxLayout fxFlex>
+            <button app-logo (click)="scrollToTop()"></button>
+          </div>
+          <form fxFlex fxLayout.lt-sm="column" [formGroup]="newslettersForm" (ngSubmit)="subscribeToNewsletters(newslettersForm)" >
```
Comment (GrandSchtroumpf) [inline libs/landing/src/lib/shell/shell.component.html:111]: rename `subscibe`
```suggestion
          <form fxFlex fxLayout.lt-sm="column" [formGroup]="newslettersForm" (ngSubmit)="subscibe(newslettersForm)" >
```

File: libs/landing/src/lib/shell/shell.component.scss [hunk 10]
```diff
@@ -12,6 +12,13 @@ landing-shell-page {
   flex-direction: column;
   height: 100vh;
 
+  button[app-logo] {
+    margin-right: 96px;
```
Comment (GrandSchtroumpf) [inline libs/landing/src/lib/shell/shell.component.scss:16]: usually margin is symetrical

File: libs/landing/src/lib/shell/shell.component.scss [hunk 11]
```diff
@@ -94,13 +94,19 @@ landing-shell-page {
         app-logo {
           margin: 0 24px 24px 0;
         }
+        article {
+          span {
+            gap: 20px 20px;
```
Comment (GrandSchtroumpf) [inline libs/landing/src/lib/shell/shell.component.scss:99]:
```suggestion
            gap: 20px;
```

**Review by GrandSchtroumpf — APPROVED** [review]

File: apps/catalog/catalog/src/app/landing/landing.component.html (hunk 1 above)
Comment (phpgeekfr) [reply apps/catalog/catalog/src/app/landing/landing.component.html:23]: created #7432

File: apps/catalog/catalog/src/app/landing/landing.component.html (hunk 2 above)
Comment (phpgeekfr) [reply apps/catalog/catalog/src/app/landing/landing.component.html:43]: created #7432

File: apps/catalog/catalog/src/app/landing/landing.component.html (hunk 3 above)
Comment (phpgeekfr) [reply apps/catalog/catalog/src/app/landing/landing.component.html:73]: created #7432

File: apps/catalog/catalog/src/app/landing/landing.component.html (hunk 4 above)
Comment (phpgeekfr) [reply apps/catalog/catalog/src/app/landing/landing.component.html:63]: created #7432

File: apps/festival/festival/src/app/event/login/login.component.html (hunk 5 above)
Comment (phpgeekfr) [reply apps/festival/festival/src/app/event/login/login.component.html:39]: created #7432

**Pull request merged**